build_stage_repository

```diff
@@ -96,9 +100,9 @@ def sync_copr_repository(collection, version, target_dir, source=False):
         'reposync',
         '--newest-only',
         '--repo',
-        f"el8-{collection}-{version}",
-        '--config',
-        'reposync_config.conf',
+        f"{dist}-{collection}-{version}",
+        '--repofrompath',
+        f"{dist}-{collection}-{version},https://download.copr.fedorainfracloud.org/results/@theforeman/{collection}-{version}-staging/rhel-{dist.replace('el', '')}-x86_64/",
```

Comment:
Just a thought: can we define `x86_64` as a parameter to the function? I'd love to start supporting arm and possibly other arches for the client repos.

Reply:
Sure, arch is a parameter now.